### DISCUSSION TITLE
Does Astro support token-weighted voting?

### DISCUSSION THREAD

**Discussion by 0xQuasark** [near-daos/astro-ui#605] [Q&A] [answered]

**Comment by 0xQuasark** [comment, marked as answer]
Not yet, but it is expected to be live with a nice UI in March. 

Check out the roadmap here: https://gov.near.org/t/2022-01-21-sputnik-astro-update/13642

**Reply by starfoxbeta** [reply]
Hi all, do you know if this is supported yet?
